src/helix-ui/styles/modules/choice-tiles.less

```diff
@@ -0,0 +1,112 @@
+@import (reference) 'modules/grid';
+
+.hxRow > label.hxChoice {
+  #grid.column();
+}
```

Comment:
This may not be needed any more. I think we added it for responsiveness, which we removed.

Reply:
this is still needed -- states get squished together if not there -- unless we wanna implement the states another way?